ci/mk-package/ubuntu/mk-package.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+chown -R `whoami` /build
+
+git --version
+stack --version
+
+echo "Cloning TorXakis"
+
+cd /build
+
+echo "Hello from docker" >> hello.txt
+
+if [ -d /build/TorXakis ]
+then
+    echo "TorXakis already cloned"
+else
+    git clone https://github.com/TorXakis/TorXakis.git
+fi
+
+cd TorXakis
+
+git fetch origin
+
+git checkout v$TXS_VERSION
+
+git pull origin v$TXS_VERSION
+
+export STACK_ROOT="/build/.stack"
+
+# We will be using a stack configuration file that is linux specific
+export STACK_YAML="stack_linux.yaml"
+
+stack build --fast
+
+export TXS_TMP=/build
+mkdir -p ${TXS_TMP}/usr/bin
+
+TXS_BIN=`stack path --local-install-root --silent --stack-yaml stack_linux.yaml`/bin
+
+cp ${TXS_BIN}/* ${TXS_TMP}/usr/bin/
+
+export CACHE_DIR="/build"
+
+echo "Looking for cvc4 in ${CACHE_DIR}/usr/bin/)"
+
+if [ -f $CACHE_DIR/usr/bin/cvc4 ] && [ -e $CACHE_DIR/usr/bin/cvc4-1.7 ]
```

Comment:
So this file must be updated when we update cvc4 and/or z3.
Added comment to https://github.com/TorXakis/TorXakis/issues/897 to remind us.

Reply:
Yeah, but the right thing to do would be to have bash functions for downloading `cvc4` and `z3` to given locations, and use them in `ci` and `ci/mk-package`. I didn't do it here, since I ran out of time, and it'd require additional testing (that I didn't break CI).